e2e/e2e.js

```diff
@@ -29,15 +29,67 @@ test('sdk init test', async () => {
 });
 
 test('test getActivities', async () => {
-  //check success response
-  // var res = await sdkClient.getXTActivityById(321)
   var res = await sdkClient.getActivities()
-  expect(res.total).toBe(9)
+  expect(res.total).toBe(10)
```

Comment:
this is a general comment on all the tests. They should not be specific to a result (unless you created the activity for the test, etc) but be generic. You would only test that the response is ok here